reader/reader_test.js

```diff
@@ -177,3 +177,30 @@ test("write to a map in a compute", function(){
 
 	QUnit.equal(map.complete, false, "value set");
 });
+
+// TODO: How can we test this? Neither onerror or onunhandledrejection catches
+// test("promise readers throw errors (#70)", function() {
```

Comment:
we have `canDev.log` and `canDev.warn`. we could create `console.error` to mimic `console` and `canLog`. (`console.error` and `console.warn` both output a stack trace)